### DISCUSSION TITLE
How to remove sample WA API Template Messages?

### DISCUSSION THREAD

**Discussion by jochenstu** [chatwoot/chatwoot#5548] [Q&A] [answered]
There are around 20 unnecessary sample message templates that a clogging up our WA channel in various languages that we don't need and no clue what to do about it. I was investigating how to remove them. What I gathered is, that [I can not remove these Message Templates](https://www.facebook.com/business/help/2047376461998278?id=2129163877102343) from inside the FB Manager. 

Is there anything that can be done about this on the chatwoot side of things? I am self hosting chatwoot on the current version, using the WA Cloud API to integrate.

**Comment by jochenstu** [comment]
Bump. If anyone else, who is self hosting chatwoot and using the wa cloud api, is not having the issue I would also like to know. Maybe this is a relic of some previous service I tried to integrate with.

<img width="2101" alt="image" src="https://user-images.githubusercontent.com/1176036/198540868-9a1d8e01-cf31-40dd-b4e1-10f700ae4a22.png">

**Comment by brunosfg** [comment]
I'm having the same issue. Found those sample templates yesterday stored in the chatwoot database. You can try to delete them and watch if they will return.

**Comment by sojan-official** [comment, marked as answer]
@jochenstu  @brunosfg The default templates for all approved WhatsApp cloud API accounts are enabled for Facebook. Chatwoot doesn't do any template management on its own. It just routinely syncs the templates from the Facebook business manager. So what you see on Chatwoot will be a replica of all approved templates in the Facebook business manager. 

tracking further enhancements over this behavior on https://github.com/chatwoot/chatwoot/issues/5246

**Comment by GKG4** [comment]
@sojan-official You mentioned routinely sync the templates, could you please let us know this sync routine?

**Reply by sojan-official** [reply]
we do it at every 5 minutes.

**Comment by Natotela** [comment]
How do I check the Sync-polling interval on a self-hosted docker config?
